### ISSUE TITLE
Column fallback to original value every time there's no translation present

### ISSUE THREAD

**Issue by benoror** [shioyama/mobility#645]
I stumbled upon ColumnFallback (https://github.com/shioyama/mobility/pull/512#issuecomment-2294408964) which mostly meets our needs, except for one small tangent:

## Expected Behavior

TL;DR; Wondering if there's a clean way to **_default to column value every time there's a missing translation_**, including for `I18n.default_locale`.

## Full Context:

At my project we currently have a huge set of tables with content that hasn't been translated, we would like to have a `default_locale` and other available languages but we're having second thoughts of the redundancy of copying over / regenerating the very same content all over again to the translation tables, hence **_we would like to default to the original value always, unless there's an explicit translation present_**, independently of the locale (as said before, even `default_locale`)

## Actual Behavior

```
column_fallback true
```

It always fallbacks to looking at the translated value except for `default_locale`

## Possible Fix

A) Workaround

Possibly using a `Proc` for the plugin value with custom logic?

B) Code feature (if ever gets contemplated):

In the original implementation I suppose the change would be somewhere around here: https://github.com/shioyama/mobility/blob/7cbca6430efe1b6fe48f5165dc99ebca0f505000/lib/mobility/plugins/active_record/column_fallback.rb#L29

...to always call `super`?

I haven't dig into the [newer implementation](https://github.com/shioyama/mobility/blob/master/lib/mobility/plugins/active_record/column_fallback.rb) tho

**Comment by benoror** [comment]
OK easiest fix for A) is as simple as:

```ruby
    column_fallback ->(locale) {
      I18n.available_locales
    }
```

this is simple enough that might not merit a full blown feature.

closing it now

**Comment by benoror** [comment]
I am so dumb, that effectively disables Mobility all together 🙈 

Going back to square one, wondering if there's a way to check wether the translation exists or not within that Proc 🤔

**Comment by mclaros** [comment]
@benoror I'm not sure I understand your question, but I'm pretty sure that lambda needs to return a boolean. A truthy result would tell Mobility to go ahead and fallback to the original column.

For example, I have a use case where I want Mobility to always fallback for English locales, regardless of region, so here's what I set:

```ruby
# NOTE: Prior to Mobility, all table column values have been in English, without
# regional differences. We want the Mobility to ignore handling 'en' content as
# translations and instead read/write to the original columns.
column_fallback -> (locale) { locale.to_s.starts_with?('en') }
```

**Comment by alepore** [comment]
I have the same problem and I'm surprised it's not a common problem.
Maybe there's something we are missing?

The issue is that, when adding Mobility to an existing codebase, I'd expect every locale to fallback to the column value that are already there (with validations etc).
Otherwise how could i use this gem?

**Comment by alepore** [comment]
TL;DR 
If `fr` fallbacks to `en`, and `en` fallbacks to the column, a missing `fr` translation should fallback on the column.

**Comment by shioyama** [comment]
I'm closing this since there is enough to cover without added features. If there is a bug somewhere please create an issue, but otherwise create/publish plugins for whatever else you need. :bowing_man:
